source/_docs/configuration.markdown

```diff
@@ -58,3 +45,20 @@ The method for running a configuration check depends on your [installation type]
 - [Configuration check on Supervised](/common-tasks/supervised/#configuration-check)
 - [Configuration check on Container](/common-tasks/container/#configuration-check)
 - [Configuration check on Core](/common-tasks/core/#configuration-check)
+
+## Reloading the configuration to apply changes
+
+For configuration changes to become effective, the configuration must be reloaded. Most integrations in Home Assistant (that do not interact with {% term devices %} or {% term services %}) can reload changes made to their configuration in `configuration.yaml` without needing to restart Home Assistant.
+
+1. There are two main ways to reload configuration changes.
+   - Under **Settings**, select the three dots menu (top right), select **Restart Home Assistant** > **Quick reload**.
+2. Another way to reload the configuration is by using the **Developer Tools**.
```

Comment:
Should we even have this in our docs? As in, the dev tools is not really meant for day2day end-users?

Reply:
Yes, I was wondering... the reason I left it is because I thought someone put it there for a reason. I'm not sure how many people use this.

Reply:
How many people still use this: I don't know either..

I think the gist is: They shouldn't need to use it anymore. There basically isn't really a reason to use it I guess

Reply:
I always use option 2 😄

I'm not in favor of removing it from the docs, it's an existing option and the user can decide which route they use.

Reply:
I'm in favor of removing **Developper tools** option from this specific procedure.
I think documentation should in general follow a minimalist approach. If there are multiple ways to get somewhere, and they have the same result, within that procedure, document one way only. (The reason I left both in was because I wasn't entirely sure if they are identical). 

This would give us one simple procedure we can link to from everywhere else where reload is needed.

Having said that I think the other option should be explained in the docs on the dev tools. If users stumble across that button and wonder what it does and how it differs from the **quick reload**, they can look it up there.

Reply:
Moving those steps to Dev tools page: https://github.com/home-assistant/home-assistant.io/pull/32449, in preparation for removing them from this section here.